mappings/net/minecraft/client/network/SrvResolver.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_6371 net/minecraft/client/network/SrvResolver
+	FIELD field_33749 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_33750 INVALID_RESOLVER Lnet/minecraft/class_6371;
+	METHOD lookupRedirect (Lnet/minecraft/class_639;)Ljava/util/Optional;
+		ARG 1 address
+	METHOD method_36909 getResolver ()Lnet/minecraft/class_6371;
```

Comment:
```suggestion
	METHOD method_36909 createSrv ()Lnet/minecraft/class_6371;
```
This creates and returns an srv-based redirector (creating a `com.sun.jndi.dns.DnsContextFactory`-delegating direction context for a redirector on each call)